cdap-docs/developers-manual/source/building-blocks/datasets/system-custom.rst

```diff
@@ -101,7 +101,14 @@ To create a Dataset of type ``UniqueCountTable``, add the following into the App
 
 You can also pass ``DatasetProperties`` as a third parameter to the ``createDataset`` method.
 These properties will be used by embedded Datasets during creation and will be available via the
-``DatasetSpecification`` passed to the Dataset constructor.
+``DatasetSpecification`` passed to the Dataset constructor. For example, to create a 
```

Comment:
OK, so I looked through the code, and it should be "OrderedTable.PROPERTY_TTL", as that is the property that  DatasetInstanceHandler.java is looking for to set the ttl.